input/fsh/au-erequesting-diagnosticrequest.fsh

```diff
@@ -10,6 +10,20 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
 
 * ^extension[http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm].valueInteger = 0
 
+* extension contains http://hl7.org/fhir/StructureDefinition/patient-sexParameterForClinicalUse named sexParameterForClinicalUse 0..1 MS
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][0].extension[code].valueCode = #SHALL:populate-if-known
+* extension[sexParameterForClinicalUse] ^extension[http://hl7.org/fhir/StructureDefinition/obligation][0].extension[actor][0].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-placer"
```

Comment:
Commenting here to get an inline comment. 
It would be very helpful to add an example to demonstrate the usage of this new extension. Please ensure a backlog item is created and linked from this PR for the addition of an example if it's not included in this PR.

Reply:
Backlog item created

Reply:
Confirmed.